src/ray/raylet/scheduling/cluster_task_manager.h

```diff
@@ -308,7 +316,7 @@ class ClusterTaskManager : public ClusterTaskManagerInterface {
   void Dispatch(
       std::shared_ptr<WorkerInterface> worker,
       std::unordered_map<WorkerID, std::shared_ptr<WorkerInterface>> &leased_workers_,
-      std::shared_ptr<TaskResourceInstances> &allocated_instances, const Task &task,
+      std::shared_ptr<TaskResourceInstances> allocated_instances, const Task &task,
```

Comment:
Hm, why remove the &?  I didn't see the `std::move` when invoking this.

Reply:
Revert  to use reference.